scipy/optimize/_highs/cython/src/HighsIO.pxd

```diff
@@ -4,13 +4,9 @@
 from libc.stdio cimport FILE
 
 cdef extern from "HighsIO.h" nogil:
-    void HighsPrintMessage(FILE* pass_output, const int level, const char* format, ...)
-
-    cdef enum HighsPrintMessageLevel:
```

Comment:
Logging/debug messaging was another large refactor area for HiGHS.  This Enum was removed entirely in favor of the new `HighsLogType`